solana/solana-ibc/programs/solana-ibc/src/storage.rs

```diff
@@ -352,12 +352,34 @@ pub fn get_provable_from<'a, 'info>(
     get(info).map_err(|err| err.with_account_name("trie"))
 }
 
+/// Used for finding the account info from the keys.
+/// Useful for finding the token mint on the source chain which cannot be derived from the denom.
+/// Would also save us some compute units to find authority and other accounts which used to be found by deriving from the seeds.
```

Comment:
Could you wrap at 80 columns? And separate first line by an empty comment so it shows as short description in rendered docs.

Reply:
Sure